apps/backend/src/modules/auth/auth.controller.ts

```diff
@@ -53,6 +66,39 @@ class AuthController extends BaseController {
 		});
 	}
 
+	/**
+	 * @swagger
+	 * /auth/authenticated-user:
+	 *   get:
+	 *     description: Return authenticated user
+	 *     security:
+	 *       - bearerAuth: []
+	 *     responses:
+	 *       200:
+	 *         description: Successfull operation
+	 *         content:
+	 *           application/json:
+	 *             schema:
+	 *               type: object
+	 *               $ref: "#/components/schemas/User"
+	 */
+	private getAuthenticatedUser(
+		options: APIHandlerOptions<{
+			user: UserDto;
+		}>,
+	): APIHandlerResponse {
+		if (!options.user) {
+			throw new AuthError({
+				message: ErrorMessage.UNAUTHORIZED,
+				status: HTTPCode.UNAUTHORIZED,
+			});
+		}
```

Comment:
Controller can't have any logic. This error is already thrown from the plugin, we don't need it